### ISSUE TITLE
ory get permission-config output is incorrect

### ISSUE THREAD

**Issue by rparsonsbb** [ory/cli#321]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/cli/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/cli/blob/master/CONTRIBUTING.md).
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [X] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Ory Network Project

https://magical-brattain-zl5vpykbux.projects.oryapis.com

### Describe the bug

```
❯ ory get permission-config --format json
{"limit":{},"namespaces":{"location":"https://storage.googleapis.com/bac-gcs-production/xxxxxxxxxx.bin"}}
```

```
❯ ory get permission-config --format yaml
limit: {}
namespaces:
  location: https://storage.googleapis.com/bac-gcs-production/xxxxxxxxxx.bin
```

So... the output is correct format :D but the actual .bin file is always just the typescript file.

version info:
```
❯ ory version
Version:    v0.2.2
Git Hash:   3ef9cd5f54d31fdc8da9ea3d9bfa1cfefb6f098d
Build Time: 2023-01-26T09:52:48Z
```

### Reproducing the bug

1. Download latest ory-cli release
2. Auth in (having to go make a password for sso was interesting, might want to document it)
3. ory use project <uuid>
4. ory get permission-config

### Version

0.2.2

### On which operating system are you observing this issue?

Linux

### In which environment are you deploying?

Ory Network

**Comment by aeneasr** [comment]
Thank you for the report! Could you clarify what your expectation of the output is?

**Comment by rparsonsbb** [comment]
I would think that given the example of exporting to a file in the [documentation](https://www.ory.sh/docs/guides/cli/configure-permission-service#export-to-file) that the output would be either yaml or json instead of a link to the file.  Is that incorrect?

**Comment by aeneasr** [comment]
I see, so you would expect the contents of `location ` to be a plan string instead of a URL?

**Comment by rparsonsbb** [comment]
No, I would think it would be either a string representing the json or yaml?

`$ ory get permission-config {project-id} --format yaml > permission-config.yaml` makes it seem like standard yaml would be the output of that command.
